src/Icicle/Data.hs

```diff
@@ -71,6 +74,14 @@ data Value =
   | Tombstone
   deriving (Eq, Show)
 
+instance PP.Pretty Value where
+  pretty v = case v of
+    StringValue t  -> PP.text (unpack t)
+    IntValue    i  -> PP.int i
+    DoubleValue d  -> PP.double d
+    BooleanValue b -> PP.pretty b
+    -- I'm too lazy
+    _              -> PP.text (show v)
```

Comment:
Yeah in general we try to avoid pattern matching on wildcards. We have (and almost certainly will) add more of these `Value` types in future